### PR TITLE
Create sacra-scripta.csl

### PR DESCRIPTION
**Pull request by johaenel**
Create Sacra Scripta Journal Citation Style of the Centre for Biblical Studies of Babeș-Bolyai University in Cluj-Napoca, Romania

### CODE REVIEW

File: sacra-scripta.csl [hunk 1]
```diff
@@ -0,0 +1,511 @@
...
+  <macro name="collection">
+    <group font-style="normal" delimiter=" " prefix="coll. ">
+      <text variable="collection-title" font-style="italic"/>
+      <text variable="collection-number"/>
+    </group>
+  </macro>
+  <macro name="locators">
+    <group>
+      <label variable="locator" form="short"/>
+      <text variable="locator" prefix=" "/>
```
Comment (POBrien333) [inline sacra-scripta.csl:236]: It's recommended to avoid unnecessary pre/suffixes and better use groups using `delimiter=" "`.

**Comment by adam3smith** [comment]
Hi @johaenel - sorry for the long wait here. I'm a bit confused by this style. I find a journal with this ISSN and title with instructions here: http://sacrascripta.reviste.ubbcluj.ro/en/ghid.php but those instructions aren't very similar for what you're submitting here, starting with the fact that they don't even allow submissions in Romanian. Could you clarify what this style is for/based on?

**Comment by adam3smith** [comment]
@johaenel are you still planning on working on this?

**Comment by adam3smith** [comment]
Closing for inactivity -- happy to re-open if you get back to us on the questions above, just post here

**Pull request closed without merging**